### PR TITLE
Add prometheus config

### PR DESCRIPTION
**Pull request by 0xmichalis**
/cc @smarterclayton @stevekuznetsov

### CODE REVIEW

File: projects/prometheus/prometheus.yml [hunk 1]
```diff
@@ -0,0 +1,165 @@
+rule_files:
```
Comment (smarterclayton) [inline projects/prometheus/prometheus.yml]: How are we going to keep this up to date every time we add something to origin?

Comment (kargakis) [reply projects/prometheus/prometheus.yml]: If this file has a canonical location, I think we can use that. We need our own `prometheus.rules` and `alertmanager.yml`.

**Comment by stevekuznetsov** [comment]
I don't have the Prometheus knowledge to review these -- @kargakis is there anything outstanding before we merge and let these soak?

**Comment by kargakis** [comment]
> @kargakis is there anything outstanding before we merge and let these soak?

Yes, we need to setup the alert manager to send emails to aos-cicd or somewhere else. I will try to make some progress on it tomorrow.

**Comment by stevekuznetsov** [comment]
Ah, ok -- I thought your other PR was for the alert-manager bits. SGTM

**Comment by kargakis** [comment]
I have updated prometheus to run as a statefulset and I found a workaround for patching `prometheus.rules`. We can add the alert manager config in a follow-up.

**Pull request closed without merging**